src/test/java/org/folio/rest/support/matchers/DomainEventAssertions.java

```diff
@@ -116,6 +118,11 @@ public static void assertUpdateEventForRequest(JsonObject oldRequest, JsonObject
     assertUpdateEvent(getLastRequestEvent(requestId), oldRequest, newRequest);
   }
 
+  public static void assertUpdateEventForCirculationRules(JsonObject oldRules, JsonObject newRules) {
+    await().until(() -> getCirculationRulesEvents().size(), greaterThan(0));
```

Comment:
Should we limit the waiting time to avoid tests running forever if something goes wrong?

Reply:
`await` awaits for 5 seconds by default